src/renderer/components/dialogs/HelpPage.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import reactStringReplace from 'react-string-replace'
+import parse from 'html-react-parser'
+import domToReact from 'html-react-parser/lib/dom-to-react'
+import DeltaDialog, { DeltaDialogBody, DeltaDialogFooter } from './DeltaDialog'
+import { Card } from '@blueprintjs/core'
+import ClickableLink from '../helpers/ClickableLink'
+import { join } from 'path'
+import fs from 'fs'
+
+const contentFilePath = join(__dirname, `../../../../static/help/__locale__/help.html`)
+
+function helpPageReadContentFile () {
+  const locale = window.localeData.locale
+  let contentFile = contentFilePath.replace('__locale__', locale)
+  if (!fs.existsSync(contentFile)) {
+    if (locale !== 'en') {
+      contentFile = contentFilePath.replace('__locale__', 'en')
+    } else {
+      throw new Error(`Cannot read HTML content, file not found: '${contentFile}'`)
+    }
+  }
+  return fs.readFileSync(contentFile, 'utf-8')
+}
+
+export function HelpPageContent ({ props }) {
+  // Make links open externaly (without this, nothing happens if one clicks on a link).
+  const parserOptions = { replace: ({ attribs, children }) => {
+    if (attribs && attribs.href && attribs.href.startsWith('http')) {
+      return (
+        <ClickableLink href={attribs.href}>{domToReact(children, parserOptions)}</ClickableLink>
+      )
+    }
+  } }
+  return parse(helpPageReadContentFile(), parserOptions)
+}
+
+function HelpPageLocalCopyHint ({ props }) {
+  const locale = window.localeData.locale
+  const localizedHref = `https://delta.chat/${locale}/help`
+  const text = window.translate('help_page_local_copy_hint')
+  const linkedText = reactStringReplace(text, /\[LINK:([^\]]+)\]/, (match, index) => (
+    <ClickableLink href={localizedHref} key={index}>{match}</ClickableLink>
+  ))
+
+  return (
+    <p style={{ textAlign: 'center', fontSize: 'smaller', borderTop: 'solid thin lightgrey', paddingTop: '0.3em' }}>
+      {linkedText}
+    </p>
+  )
+}
+
+export default function HelpPage (props) {
+  return (
+    <DeltaDialog
+      title={window.translate('help_page_title')}
```

Comment:
The menu item in german says 'Hilfe' but the dialog title is 'Frequently asked Questions'. I would like to have it consistent, but probably this is a translation fault?

Reply:
yes, i would also call it "Help" in the app everywhere.